interface-definitions/dns-forwarding.xml.in

```diff
@@ -188,6 +188,178 @@
                 </properties>
                 <defaultValue>0.0.0.0 ::</defaultValue>
               </leafNode>
+              <node name="advanced">
```

Comment:
> For me, it is not clear which format we should use for PTR names.

I don't know if my understanding is correct. If my understanding is correct, PTR records are named in a tree format similar to domain name converted from specific encoded IPv4 / IPv6, and the resolution value is the real domain name.

It is suggested to modify the command to the following format:

```
set service dns forwarding advanced zone localhost.local a local.loc value '192.0.2.1'
set service dns forwarding advanced zone localhost.local mx mx5.local.loc value 'mx10-foo.local.loc'
set service dns forwarding advanced zone localhost.local ptr 4.2.0.192.in-addr.arpa. value '192.0.2.4'
set service dns forwarding advanced zone localhost.local ptr 2.2.0.192.in-addr.arpa. value '192.0.2.2'
set service dns forwarding advanced zone localhost.local ptr 8.2.0.192.in-addr.arpa value '192.0.2.8'
set service dns forwarding allow-from '100.64.0.0/24'
set service dns forwarding allow-from '192.0.2.0/24'
set service dns forwarding allow-from '192.168.122.0/24'
set service dns forwarding allow-from '127.0.0.1/32'
set service dns forwarding listen-address '100.64.0.1'

```